packages/core/src/sso/types/oidc.ts

```diff
@@ -57,7 +57,8 @@ export const oidcTokenResponseGuard = z.object({
   id_token: z.string(),
   access_token: z.string().optional(),
   token_type: z.string().optional(),
-  expires_in: z.number().optional(),
+  // Microsoft EntraID may return string type for expires_in
+  expires_in: z.number().or(z.string()).optional(),
```

Comment:
not sure whether we should convert the value to number when it is of string type.

Reply:
not being used sofar